Makefile

```diff
@@ -58,7 +58,6 @@ verify:
 	hack/verify-gofmt.sh ||r=1;\
 	hack/verify-govet.sh ||r=1;\
 	hack/verify-imports.sh ||r=1;\
-	hack/verify-codegen.sh ||r=1;\
```

Comment:
Can the `hack/verify-codegen.sh` be removed since it's not used anywhere anymore ?

Reply:
It probably should not, it should be used to validate that things have been properly generated.